src/widgets/media-requests/MediaRequestStatsTile.tsx

```diff
@@ -36,41 +45,44 @@ function MediaRequestStatsTile({ widget }: MediaRequestStatsWidgetProps) {
   }
 
   return (
-    <Flex gap="md" wrap="wrap">
-      <Card w={100} h={100} withBorder>
-        <Center h="100%">
-          <Stack spacing={0} align="center">
-            <Text>
-              {data.filter((x) => x.status === MediaRequestStatus.PendingApproval).length}
-            </Text>
-            <Text color="dimmed" align="center" size="xs">
-              {t('stats.pending')}
-            </Text>
-          </Stack>
-        </Center>
+    <Flex
+      w="100%"
+      h="100%"
+      gap="md"
+      direction={ widget.properties.direction != 'vertical' ? 'row' : 'column' }
+    >
+      <Card w="100%" h="100%" withBorder style={{flex:"1 1 auto"}}>
+        <Stack w="100%" h="100%" align="center" justify="center" spacing={0}>
+          <Text align="center">
+            {data.filter((x) => x.status === MediaRequestStatus.PendingApproval).length}
+          </Text>
+          <Text color="dimmed" align="center" size="xs">
+            {t('stats.pending')}
+          </Text>
+        </Stack>
       </Card>
```

Comment:
Here we could also make a component with a value and label property and reuse that